engine/runtime-compiler/src/main/java/org/enso/compiler/docs/DocsUtils.java

```diff
@@ -76,20 +76,36 @@ static String toFqnOrSimpleName(Name ir) {
   }
 
   static String toSignature(DefinitionArgument a) {
-    var sb = new StringBuilder();
-    if (a.suspended()) {
-      sb.append("~");
-    }
-    var name = a.name().name();
-    sb.append(name);
-    if (!ConstantsNames.SELF_ARGUMENT.equals(name)) {
-      var type = extractTypeOrAny(a);
-      sb.append(":").append(type);
+    if (isBlankArg(a)) {
+      return "_";
+    } else {
+      var sb = new StringBuilder();
+      if (a.suspended()) {
+        sb.append("~");
+      }
+      var name = a.name().name();
+      sb.append(name);
+      if (!ConstantsNames.SELF_ARGUMENT.equals(name)) {
+        var type = extractTypeOrAny(a);
+        sb.append(":").append(type);
+      }
+      if (a.defaultValue().isDefined()) {
+        sb.append("=");
+      }
+      return sb.toString();
     }
-    if (a.defaultValue().isDefined()) {
-      sb.append("=");
+  }
+
+  /** Returns true if the argument represents an underscore (blank) argument. */
+  private static boolean isBlankArg(DefinitionArgument arg) {
```

Comment:
There is no better way to check if the `DefinitionArgument` was `_`. This information is simply lost in the compilation pipeline.